e2e/tests/myatb.e2e.ts

```diff
@@ -0,0 +1,184 @@
+import {device, by, element, expect} from 'detox';
+import {findTextViewElement, goToPage, scrollToElement, scrollToElementText} from "../utils/common";
+import {skipOnboarding} from "../utils/onboarding";
+import {addFavouriteLocation, deleteFavouriteLocation, editFavouriteLocation, sortLabelExists} from "../utils/myatb";
+import setLocation from "../utils";
+
+describe('My AtB', () => {
+    beforeAll(async () => {
+        await device.launchApp({
+            permissions: {
+                location: 'inuse',
+            },
+            languageAndLocale: {
+                language: 'en',
+                locale: 'US',
+            },
+        });
+        await setLocation(62.4305, 9.3951);
+        await skipOnboarding();
+
+        await goToPage('My AtB')
+    });
+
+    beforeEach(async () => {
+        //await device.reloadReactNative();
+        await goToPage('My AtB')
+        await element(by.type('RCTCustomScrollView')).scrollTo('top')
```

Comment:
Her kan det være lik så greit å bare reload, og så gå til Profil-tabben, sannsynligvis kjappere enn å scrolle opp

```suggestion
        await device.reloadReactNative();
        await goToPage('My AtB')
```

Reply:
Testet dette nå, og det tok faktisk ganske mye lengre tid å reloade for så å gå til tabben. Opplever under test at relaod stjeler en del tid. Det var derfor jeg tenkte at valgte løsning var bedre så lenge tilstand før hver test ble lik.